CZI_mpl.tex

```diff
@@ -55,7 +55,7 @@ \section{Goals}
 
 
 % The goal is the maintenance; curating is the method of achieving it.
-\subsection{Maintenance of the library}
+\subsection{Revitalizing the developer community}
```

Comment:
"Revitalizing" implies a lack of vitality, which I think is misleading, or at least not the tone we want to convey.  Maybe "Enlarging"? Or "Cultivating"?  The point of the latter is that we need more efficient and effective development, and this is far from being just a matter of size (number of contributors).

Reply:
I like cultivating. My point here is that “Maintenance” might not sound like something they want to fund.  OTOH if other proposals have been successful proposing maintenance as a big part of the proposal, then maybe this concern is misplaced

Reply:
CZI specifically calls out maintenance in the program description:  "The goal of the program is to support software maintenance, growth, development, and community engagement for these critical tools."  and this funding path was conceived in part to address the gap left by other funding sources.

Reply:
OK, "Maintenance and Growth of the Library"?

Reply:
"Maintenance and Growth of Library and Community" ?